Documentation/dev/developer-guide.md

```diff
@@ -27,6 +27,23 @@ $ brew install go kubernetes-helm make docker dep coreutils jq faq python@3
 $ pip3 install operator-courier
 ```
 
+## Setup
+
+It is important that you clone the project to the right location in your $GOPATH.
+
+The place this repo must be located at is:
+
+```
+$GOPATH/src/github.com/operator-framework/operator-metering
+```
+
+If you're checking out the repo for the first time, you can run this to clone the repo to the correct location:
```

Comment:
"When cloning this repository, you can run the following to ensure that the project files are stored in the right location: ..."